src/transport.ts

```diff
@@ -183,13 +184,30 @@ export class WebRTCDirectTransport implements Transport {
       }
     }
 
+    const eventListeningName = isFirefox ? 'iceconnectionstatechange' : 'connectionstatechange'
```

Comment:
Good point, well `iceconnectionstatechange` event has been supported since [Firefox 24](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/iceconnectionstatechange_event) which is about a decade old, and `connectionstatechange` has been supported for [at least two years](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/connectionstatechange_event) in all other browsers, so I would say this is low risk.